VMEncryption/main/handle.py

```diff
@@ -138,7 +138,8 @@ def disable_encryption():
             if crypt_item.dev_path.startswith("/dev/sd"):
                 logger.log('Updating crypt item entry to use mapper name')
                 logger.log('Device name before update: {0}'.format(crypt_item.dev_path))
-                crypt_item.dev_path = disk_util.query_dev_id_path_by_sdx_path(crypt_item.dev_path)
+                # crypt_item.dev_path = disk_util.query_dev_id_path_by_sdx_path(crypt_item.dev_path)
```

Comment:
probably no need to comment this, it's in version control

Reply:
Good point, I will go ahead and remove it.